content/blog/2024-010-15-introduce-r3-to-recognition-restful-url/data_interaction_flow.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Introduce R3 to recognition RESTFul URL"
+date: 2024-10-17
+author: Han Liu
+description: "This article will introduce how to use R3 to identify RESTFul URLs and integrate them into SkyWalking."
+tags:
+- RESTFul
+- R3
+---
+
+## Background
+
+In modern applications, services are typically provided through RESTFul HTTP URLs.
+Using RESTFul HTTP URLs (as unique resource identifiers) offers high readability, making it easier for both clients and servers to understand.
+However, in the observability field, this approach poses several challenges:
+
+1. **A large number of endpoints (HTTP URL)**: Browsing through all externally provided endpoints becomes more difficult, making it hard to identify problematic endpoints.
```

Comment:
For SkyWalking, only URI is considered as endpoint.

Reply:
Done